README.md

```diff
@@ -1,23 +1,42 @@
 # UE4SSCPPTemplate
 
-To setup your project, run the following command in this directory, replacing "YourModName" with the name of your mod (no spaces or special characters, please!):
+This repository includes some custom `xmake` tasks to assist in your mod development.
 
-```bash
-.\new_mod_setup.bat YourModName
-```
+- `xmake newmod`
+- `xmake installmod`
+- `xmake bi`
+- `xmake ue4ss`
 
-This will create a directory vsxmake2022 with a `.sln` file that you can open.
+## `xmake newmod [-r, --regen] name`
 
-Then, when you are ready to build and install your mod into your game, run the following command in this directory:
+The `newmod` command will ensure you have the UE4SS repository cloned and then bootstrap new mod creation.
 
-```bash
-.\build_and_install_mod.bat YourModName "Path\To\Your\Game\UE4SS\Install\Directory" Build__Configuration 
-```
+Running `xmake newmod CoolMod` will generate `/Mods/CoolMod/...` in your repository and automatically link it with the xmake system. If you have VS2022 installed then the `newmod` command will also automatically generate a VS project which can be located at `/vsxmake2022/UE4SSCppTemplate.sln`.
 
-For example:
+## `xmake installmod [-d, --exedir] name`
 
-```bash
-.\build_and_install_mod.bat MyAwesomeMod "C:\Program Files (x86)\Steam\steamapps\common\Deep Rock Galactic\FSD\Binaries\Win64" Game__Shipping__Win64
-```
+The `installmod` command will copy your build output to a specific game directory. The configuration of the build you want to install is derived from the current xmake configuration. If you want to specify a specific configuration to install, then you can change the xmake configuration by running `xmake f -m "Game__Debug__Win64"`.
 
-If you want to delete a mod from your project, you can just delete the mod's folder and its entry in the root xmake.lua file.
\ No newline at end of file
+Running `xmake installmod --exedir="Path\To\Your\Game\Dir" CoolMod` will copy the build output and install `CoolMod` in the correct subdir in the game's mod folder.
+
+Note that running `xmake installmod` does not automatically build your mod. You have to manually build it with `xmake build` or use the `xmake bi` task to build and install your mod.
+
+## `xmake bi [-d, --exedir] name`
+
+This is a shorthand task that will run `xmake build` followed by the `xmake installmod` command. This one-liner command facilitates rapid testing/iterating of your mod.
```

Comment:
`xmake build` should be `xmake build -y` (I think this is an important distinction to make in-case someone wants to run xmake build manually)